spec/spec_helper.rb

```diff
@@ -93,3 +87,11 @@
   #   # as the one that triggered the failure.
   #   Kernel.srand config.seed
 end
+
```

Comment:
Why was this commented out and moved?  simplecov has to be before all library files are loaded.